Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Jiva-Memory-Consumption/memory-consumption

```diff
@@ -111,47 +60,12 @@ python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Ch
 exit 1;
 fi
 
-echo "********Deprovisioning Busybox Application*******"
-
-test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-deprovision-mem-check-jiva metadata="")
-echo $test_name
-cd litmus
-
-################
-# LitmusBook 3 #
-################
+#########################################
+# Deprovisioning test related resources #
+#########################################
 
-: << EOF
-  --------------------------------------------------------------------------------------------------------------------     
- | specAttribute     | kind   |         default value               | test specifc value                              |
-  ------------------------------------------------------------------------------------------------------------------- |
- | appLabel          | env    | app=busybox-sts                     | app=busybox-mem-jiva                            |
- | pvcName           | env    | openebs-busybox                     | busybox-ctrl-kill                               | 
- | appNamespace      | env    | app-busybox-ns                      | mem-check-jiva                                  | 
- | storage clas      | env    | openebs-cstor-sparse                | openebs-jiva-default                            |
- | deploy type       | env    | statefulset                         | deployment                                      |
- | Action            | env    | value: provision                    | value: deprovision                              |
- | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: busybox-mem-check-deprovision-jiva|
- | Litmus job label  | label  | app: busybox-litmus                 | busybox-deprovision-mem-check-jiva              |
-  ---------------------------------------------------------------------------------------------------------------------
-EOF
-
-cp apps/busybox/deployers/run_litmus_test.yml deprovision_mem_check_jiva.yml
-
-sed -i -e 's/generateName: litmus-busybox-deploy/generateName: busybox-mem-check-deprovision-jiva/g' \ 
--e 's/app: busybox-litmus/app: busybox-deprovision-mem-check-jiva/g' \
--e 's/app=busybox-sts/app=busybox-mem-jiva/g' \
--e 's/value: statefulset/value: deployment/g' \
--e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
--e 's/value: app-busybox-ns/value: mem-check-jiva/g' \
--e 's/value: provision/value: deprovision/g' deprovision_mem_check_jiva.yml
-
-cat deprovision_mem_check_jiva.yml
-
-bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-deprovision-mem-check-jiva' job=deprovision_mem_check_jiva.yml
-cd ..
-bash Openshift-EE/utils/dump_cluster_state;
-bash Openshift-EE/utils/event_updater jobname:memory-consumption-jiva $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+kubectl delete pod --all -n memleak-jiva 
+kubectl delete pvc --all -n memleak-jiva
```

Comment:
@shashank855 Can you do this in litmus itself?

Reply:
@gprasath I have made the changes.

Reply:
Thanks @shashank855